labs/iot/gps-lab/README.md

```diff
@@ -0,0 +1,53 @@
+# GPS Lab
+
+This folder contains a lab which displays the GPS coordinate in an Azure Map in real time. It collects the GPS coordinates from a GPS sensor attached to a Raspberry Pi and transmits the data
+to IoT hub. The web application containing the azure map reads the data from IoT Hub and displays on the map.
+
+| Author | [Jim Bennett](https://github.com/JimBobBennett) |
+|:---|:---|
+| Target platform   | <ul><li>Raspberry Pi</li><li>Python with Flask</li></ul><i>Optionally you can run this on a PC or Mac using a virtual IoT device</i> |
+| Hardware required | If you are running this on a Raspberry Pi:<ul><li>[GPS Module Receiver](https://www.amazon.com/Navigation-Positioning-Microcontroller-Compatible-Sensitivity/dp/B084MK8BS2)</li><li>Raspberry Pi 4</li><li>Micro SD Card</li><li>An SD card to USB converter that matches the USB ports on your device if your device doesn't have an SD card slot</li><li>Raspberry Pi 4 power supply (USB-C)</li><li>keyboard, mouse and monitor</li><li>[micro-HDMI to HDMI adapter or cable](https://www.raspberrypi.org/products/micro-hdmi-to-standard-hdmi-a-cable/)</li></ul> |
+| Software required | <ul><li>[Visual Studio Code](http://code.visualstudio.com?WT.mc_id=iotcurriculum-github-jabenn)</li><li> OR [Visual Studio](https://visualstudio.microsoft.com/)</li></ul><br>If you are running this on a Raspberry Pi:<ul><li>[Raspberry Pi Imager](https://www.raspberrypi.org/downloads/)</li></ul>*There are optional installs for Windows and Linux that you may need to install later to connect to the Pi, depending on which version of the OS you are using.* |
+| Azure Services | <ul><li>[Azure IoT Hub](https://azure.microsoft.com/en-us/services/iot-hub/?WT.mc_id=iotcurriculum-github-jabenn)</li><li>[Azure Maps](https://azure.microsoft.com/en-us/services/azure-maps/?WT.mc_id=iotcurriculum-github-jabenn)</li></ul> |
```

Comment:
URLs should not contain locale information - that way they will redirect to the users locales instead of staying on one specified in the URL. For example, https://azure.microsoft.com/en-us/services/iot-hub/ should be https://azure.microsoft.com/services/iot-hub/ without the en-us locale